src/main/java/com/airlines/dao/CrewDaoImp.java

```diff
@@ -0,0 +1,101 @@
+package com.airlines.dao;
+
+import com.airlines.entity.Citizenship;
+import com.airlines.entity.Crew;
+import com.airlines.entity.CrewMember;
+import com.airlines.entity.Position;
+import com.airlines.exception.DaoOperationException;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CrewDaoImp implements CrewDao {
+    private static final String INSERT_CREWS_CREW_MEMBERS = "INSERT INTO crews_crew_members (crews_id, crew_members_id) VALUES(?, ?);";
+    private static final String SELECT_CREW_MEMBERS_BY_CREW__ID =
+    "SELECT * FROM crew_members  " +
+            "LEFT JOIN crews_crew_members " +
+            "ON crew_members.id = crews_crew_members.crews_members_id " +
+            "WHERE crews_crew_members.crews_id=?;";
+
+    private static final String SELECT_CREW_MEMBERS_BY_CREW__NAME =
+               "SELECT *" +
+               "FROM crew_members " +
+               "LEFT JOIN (crews_crew_members JOIN crew USING (crew_name))" +
+               "USING (crew_members_id) WHERE crew_name=?;";
+
+    private final DataSource dataSource;
+
+    public CrewDaoImp(DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+    public void addMemberToCrew(CrewMember crewMember, Crew crew) {
+        try (Connection connection = dataSource.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(INSERT_CREWS_CREW_MEMBERS)) {
+            preparedStatement.setInt(1, crew.getId());
+            preparedStatement.setInt(2, crewMember.getId());
+        } catch (SQLException e) {
+            throw new DaoOperationException("Couldn't save  CrewMember " + crewMember, e);
+        }
+    }
+
+    public List<CrewMember> findCrewMembersById(int idCrew) {
+        List<CrewMember> list;
```

Comment:
Please, declare variables in places they use:

public List<CrewMember> findCrewMembersById(int crewId) {
 try (Connection connection = dataSource.getConnection();
             PreparedStatement preparedStatement = connection.prepareStatement(SELECT_CREW_MEMBERS_BY_CREW__ID)) {
            List<CrewMember> = new ArrayList<>();
            preparedStatement.setInt(1, crewId);
            ResultSet resultSet = preparedStatement.executeQuery();
            while (resultSet.next()) {
                CrewMember crewMember = parseRow(resultSet);
                list.add(crewMember);
            }
            return list;

Reply:
fixed in ed36dce27a52f51b2246b22e6eb254afa9277071